feat: Adds a smarter default for `in_project` (Ready for Review) (#119)
jarednorman/bugsnag-elixir: README.md

If a sanitizer function throws an exception while running, it will log out a warning and return the string `[CENSORED DUE TO SANITIZER EXCEPTION]`

### `application`

**Default:** `nil`

> If you want a smart default for the `in_project`, you can set the `application`
> value. Set this to the name of your application with the following:

Reviewer: ```suggestion
Sets the `application` name to generate a smart default for the `in_project` value. Set this to the name of your application with the following:
```
